regenerate.sh

```diff
@@ -24,29 +24,95 @@ trap finish EXIT
 
 export GOBIN=${WORKDIR}/bin
 export PATH=${GOBIN}:${PATH}
-mkdir -p ${GOBIN}
+mkdir -p "${GOBIN}"
```

Comment:
It's always good to quote variables. If you want to make that change, I would suggest making that change every other place as well. 

Also see https://google.github.io/styleguide/shellguide.html#quoting